### ISSUE TITLE
[MenuItem]: A11y Screen reader announces name twice

### ISSUE THREAD

**Issue by AlekseiSch** [SAP/ui5-webcomponents#11405]
### Bug Description

When users interact with the MenuItem component using screen readers such as JAWS and NVDA, the menu item's name is announced twice. For example:
`New File New File undefined 1 of 6`
Additionally, even when an accessibleName is explicitly specified, the screen reader continues to announce the name twice (though it removes the "undefined"). For example:
`New File New File 1 of 6`

### Affected Component

MenuItem

### Expected Behaviour

The screen reader should announce the menu item name only once, in the following format:
`New File 1 of 6`

### Isolated Example

https://sap.github.io/ui5-webcomponents-react/v2/?path=/docs/modals-popovers-menu--docs

### Steps to Reproduce

1. Open a menu implemented with the MenuItem component https://sap.github.io/ui5-webcomponents-react/v2/?path=/docs/modals-popovers-menu--docs
2. Interact with the menu using screen readers (JAWS or NVDA).
3. Listen to the announced text.

### Log Output, Stack Trace or Screenshots

![Image](https://github.com/user-attachments/assets/f2bf5c5e-46cf-43eb-978b-7bec9e18abd6)

### Priority

None

### UI5 Web Components Version

2.9.2

### Browser

Chrome

### Operating System

Windows

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by TeodorTaushanov** [comment]
Hello @SAP/ui5-webcomponents-topic-b, can you check this issue?
